blazor/dialog/predefined-dialogs.md

```diff
@@ -0,0 +1,338 @@
+
+{% tabs %}
+
+{% highlight tabtitle="alert.razor" %}
+{% include_relative code-snippet/alert-dimension.razor %}
+{% endhighlight %}
+
+{% highlight tabtitle="confirm.razor" %}
+{% include_relative code-snippet/confirm-dimension.razor %}
+{% endhighlight %}
+
+{% highlight tabtitle="prompt.razor" %}
+{% include_relative code-snippet/prompt-dimension.razor %}
+{% endhighlight %}
+
+{% endtabs %}
+
+![Alert dimension Dialog](./images/blazor-alert-dimension.png)
+
+![confirm dimension Dialog](./images/blazor-confirm-dimension.png)
+
+![prompt dimension Dialog](./images/blazor-prompt-dimension.png)
+
+## Close Button Dialog
+
+You can customize the close icon using [ShowCloseIcon](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Popups.SfDialog.html#Syncfusion_Blazor_Popups_SfDialog_ShowCloseIcon) property. If the `ShowCloseIcon` property is set to `true` then the close icon will appear. Use the following code to enable the `ShowCloseIcon`.
+
+{% tabs %}
+
+{% highlight tabtitle="alert.razor" %}
+{% include_relative code-snippet/alert-close-button.razor %}
+{% endhighlight %}
+
+{% highlight tabtitle="confirm.razor" %}
+{% include_relative code-snippet/confirm-close-button.razor %}
+{% endhighlight %}
+
+{% highlight tabtitle="prompt.razor" %}
+{% include_relative code-snippet/prompt-close-button.razor %}
+{% endhighlight %}
+
+{% endtabs %}
+
+![Alert close icon Dialog](./images/blazor-alert-close-button.png)
+
+![Confirm close icon Dialog](./images/blazor-confirm-close-button.png)
+
+![Prompt close icon Dialog](./images/blazor-prompt-close-button.png)
+
+## Customization of Action Buttons
+
+You can customize the predefined dialog buttons by using [PrimaryButtonOptions](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Popups.DialogButton.html#Syncfusion_Blazor_Popups_DialogButton_IsPrimary) property. Use the following code to customize the predefined dialog buttons.
+
+{% tabs %}
+
+{% highlight tabtitle="alert.razor" %}
+{% include_relative code-snippet/alert-action-button.razor %}
+{% endhighlight %}
+
+{% highlight tabtitle="confirm.razor" %}
+{% include_relative code-snippet/confirm-action-button.razor %}
+{% endhighlight %}
+
+{% highlight tabtitle="prompt.razor" %}
+{% include_relative code-snippet/prompt-action-button.razor %}
+{% endhighlight %}
+
+{% endtabs %}
+
+![Alert action buttons Dialog](./images/blazor-alert-action-button.png)
+
+![Confirm action buttons Dialog](./images/blazor-confirm-action-button.png)
+
+![Prompt action buttons Dialog](./images/blazor-prompt-action-button.png)
+
+## Customization of Dialog Content
+
+You can customize the predefined dialogs using [DialogTemplates.childContent](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Popups.DialogTemplates.html#Syncfusion_Blazor_Popups_DialogTemplates_Content) property. Use the following code, to render the textbox component inside `Prompt` dialog.
```

Comment:
You can customize the predefined dialogs using the [DialogTemplates.childContent](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Popups.DialogTemplates.html#Syncfusion_Blazor_Popups_DialogTemplates_Content) property. Use the following code to render the textbox component inside `Prompt` dialog: